frontend/src/pages/Unauthorized.tsx

```diff
@@ -1,26 +1,60 @@
-import { styled } from "@mui/material/styles";
+import { Grid, Typography, Button, styled } from "@mui/material";
```

Comment:
this code should have already been merged into development, i suspect you are getting ahead of yourself, with too many pr's on the go. is this not the same code that i approved in this pr https://github.com/bcgov/gdx-agreements-tracker/pull/648

Reply:
you are correct... too much on the go.  Working on figuring out a system.